examples/cov2d_simulation.py

```diff
@@ -54,20 +56,23 @@
 filters = [RadialCTFFilter(pixel_size, voltage, defocus=d, Cs=2.0, alpha=0.1)
            for d in np.linspace(defocus_min, defocus_max, defocus_ct)]
 
-# Create a simulation object with specified filters
+# Create a simulation object with specified filters and 3D map of a 70S Ribosome
+logger.info('Load 3D map and creat simulation object.')
+infile = mrcfile.open(os.path.join(DATA_DIR, 'clean70SRibosome_vol_65p.mrc'))
+vols = infile.data
+vols = vols[..., np.newaxis]
+
 sim = Simulation(
     n=num_imgs,
+    vols=vols,
     C=num_maps,
     filters=filters
 )
 
-# Load 3D map from the data file, corresponding to the experimentally obtained EM map of a 70S Ribosome.
-logger.info('Load 3D map and downsample to desired grids.')
-vols = np.load(os.path.join(DATA_DIR, 'clean70SRibosome_vol.npy'))
-vols = vols[..., np.newaxis]
-
-# Downsample the 3D map to 8X8X8
-vols = downsample(vols, (img_size*np.ones(3, dtype=int)))
+# Downsample the 3D map to desired resolution This can be done by the internal function of sim object.
+# Below we use alternative implementation to obtain the exact result with Matlab version.
+logger.info(f'Downsample 3D map to desired grids of {img_size} x {img_size} x {img_size}.')
+vols = downsample(sim.vols, (img_size*np.ones(3, dtype=int)))
```

Comment:
Why can't we downsample before creating the `Simulation` object?

Reply:
Moved in the new commit 88cd518.